PSCompatibilityCollector/Microsoft.PowerShell.CrossCompatibility/Collection/TypeDataCollector.cs

```diff
@@ -106,7 +187,7 @@ public static class TypeDataConversion
 
             Type[] types = asm.GetTypes();
             JsonDictionary<string, JsonDictionary<string, TypeData>> namespacedTypes = null;
-            if (types.Any())
+            if (types.Length > 0)
```

Comment:
in line 193, why not use `types` from line 188?

Reply:
Good catch!